projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -53,4 +54,11 @@ public static function load_coming_soon() {
 			require_once __DIR__ . '/features/coming-soon/coming-soon.php';
 		}
 	}
+
+	/**
+	 * Load WP REST API plugins for wpcom
+	 */
+	public static function load_wpcom_rest_api_endpoints() {
+		wpcom_rest_api_v2_load_plugin_files( 'features/wpcom-endpoints/*.php' );
```

Comment:
Should we check the function is defined first, just in case someone on Atomic has somehow prevented Jetpack from loading? Or is that already not a thing that can ever happen?

Reply:
If Jetpack has somehow been prevented from loading, we probably have other problems, but it won't hurt to check